clients/src/test/java/org/apache/kafka/clients/producer/MockProducerTest.java

```diff
@@ -114,7 +114,7 @@ public void testManualCompletion() throws Exception {
 
         Future<RecordMetadata> md3 = producer.send(record1);
         Future<RecordMetadata> md4 = producer.send(record2);
-        assertTrue(!md3.isDone() && !md4.isDone(), "Requests should not be completed.");
+        assertFalse(md3.isDone() && !md4.isDone(), "Requests should not be completed.");
```

Comment:
@dengziming hello , Is it the same condition?

Reply:
Sorry, I find this is not right, just add another pr to fix it.